doc/signify.txt

```diff
@@ -200,17 +201,24 @@ This loads Sy, but it won't look for changes. You can toggle it anytime via
 :SignifyToggle.
 
 ------------------------------------------------------------------------------
-                                                       *g:signify_skip_filetype*
+                                               *g:signify_skip_filename_pattern*
                                                        *g:signify_skip_filename*
+                                                       *g:signify_skip_filetype*
 >
     let g:signify_skip_filetype = { 'vim': 1, 'c': 1 }
     let g:signify_skip_filename = { '/home/user/.vimrc': 1 }
 <
-Don't activate the plugin for these filetypes and/or filenames.
+Don't activate the plugin for these filetypes and/or filenames. Filenames have
+to be absolute paths.
 
-NOTE: Filenames have to be absolute paths.
+These options must be |Dict|s for faster lookup.
+>
+    let g:signify_skip_filename_pattern = [ 'foo.*bar', 'tmp' ]
+<
+Don't activate the plugin for filenames matching these patterns. Internally
+this uses |match()|.
```

Comment:
Not sure why it's worth calling out the use of `match()` here.  It's a pattern match.  People using this option should know how pattern matches work.  `match()` doesn't behave any differently in that regard.